### PR TITLE
Remove the limit from string columns in all tables

### PR DESCRIPTION
**Pull request by carbonin**
In older versions of activerecord "t.string" columns without an explicit limit were limited to 255 by default.

In newer versions that limit has been removed.

This is creating a discrepancy between databases which have been migrated through the older versions of activerecord and ones which were newly created.

This causes difficult to diagnose bugs for no reason.

https://bugzilla.redhat.com/show_bug.cgi?id=1510605

### CODE REVIEW

**Comment by carbonin** [comment]
@miq-bot assign @Fryguy 
@miq-bot add_label bug

/cc @yrudman

**Comment by carbonin** [comment]
@Fryguy are we considering backporting this or should @yrudman create a PR to truncate the string for the error in the BZ?

**Comment by yrudman** [comment]
@Fryguy PR truncating specific string for above BZ: https://github.com/ManageIQ/manageiq/pull/16444

File: manageiq-schema.gemspec [hunk 1]
```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency "more_core_extensions"
   s.add_dependency "pg", "~> 0.18.2"
   s.add_dependency "rails", "~> 5.0.2"
+  s.add_dependency "pg-pglogical", "~> 2.0.0"
```
Comment (Fryguy) [inline manageiq-schema.gemspec]: Why is this needed?

Comment (carbonin) [reply manageiq-schema.gemspec]: It's not! Looks like I got bit by switching branches around ...

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit e42454fc4034969a5edeaa592eb03b6c428da78e
Author: Jason Frey <fryguy9@gmail.com>
Date:   Thu Nov 16 11:21:46 2017 -0500

    Merge pull request #125 from carbonin/remove_limit_from_string_columns
    
    Remove the limit from string columns in all tables
    (cherry picked from commit b8206af536e5b747a748ca451619f4dd72e4a1ae)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1513123
```
